src/specklepy/objects/GIS/geometry.py

```diff
@@ -12,14 +12,24 @@
 )
 
 
-class GisPolygonGeometry(
-    Base, speckle_type="Objects.GIS.PolygonGeometry", detachable={"displayValue"}
-):
+class PolygonGeometry(Base, speckle_type="Objects.GIS.PolygonGeometry"):
     """GIS Polygon Geometry"""
 
-    boundary: Optional[Union[Polyline, Arc, Line, Circle, Polycurve]] = None
-    voids: Optional[List[Union[Polyline, Arc, Line, Circle, Polycurve]]] = None
-    displayValue: Optional[List[Mesh]] = None
+    boundary: Optional[Polyline]
+    voids: List[Polyline]
+
+    def __init__(
+        self,
+        units: Optional[str] = None,
+        boundary: Optional[Polyline] = None,
+        voids: Optional[List[Polyline]] = None,
+    ) -> None:
```

Comment:
ideally all these should be Required, but we need to have an option for an empty constructor because of the serializer design. On Receive, objects are being initialized with empty constructors first